docker-compose.test.yaml

```
@@ -0,0 +1,44 @@
+version: "3.9"
+services:
+  toxiproxy:
+    image: ghcr.io/shopify/toxiproxy:2.11.0
+    hostname: toxiproxy
+
+  unit_test:
+    # The tests might require permissions to write to coverage out files.
+    # If you encounter issues with permissions, you can try running the container as root.
+    user: root
+    image: cimg/go:1.22.5
+    working_dir: /etc/ocpp-go
+    volumes:
+      - .:/etc/ocpp-go:rw
+    command:
+      - /bin/bash
+      - -c
+      - |
+        go test -v -covermode=count -coverprofile=ocppj.out ./ocppj
+        go test -v -covermode=count -coverprofile=ocpp16.out -coverpkg=github.com/lorenzodonini/ocpp-go/ocpp1.6/... github.com/lorenzodonini/ocpp-go/ocpp1.6_test
+        go test -v -covermode=count -coverprofile=ocpp201.out -coverpkg=github.com/lorenzodonini/ocpp-go/ocpp2.0.1/... github.com/lorenzodonini/ocpp-go/ocpp2.0.1_test
+        sed '1d;$d' ocppj.out >> coverage.out
```

Coveralls upload is failing because of this, as it skips the `mode: count` line (required).

Easiest fix:
```suggestion
        sed '0d;$d' ocppj.out >> coverage.out
```